### PR TITLE
Refactor some class-find-way with DexKit.java instead of ConfigTable.kt

### PR DESCRIPTION
**Pull request by NextAlone**
Signed-off-by: NextAlone <12210746+NextAlone@users.noreply.github.com>

# Refactor some class-find-way with DexKit.java instead of ConfigTable.kt

## 更改内容 / Types of Changes

- [ ] 内核 Core
- [ ] 修复 Bugfix
- [ ] 新功能 New feature
- [x] 增强/优化 Enhancement/optimization
- [ ] 文档 Documentation
- [ ] UI界面 User interface

## 由这个PR解决或关闭的Issues / Issues Fixed or Closed by This PR

-

## 重构进度 / Progress

- 与重构主线无关

## 检查列表 / Checklist

- [x] 我的代码符合本仓库的代码规范 My code follows the code style of this project
- [ ] 我已经测试了这些更改，它们可以正常工作，且不会破坏任何功能(或我可以解决) I have tested the changes and verified that they work and don't break anything (as well as I can manage).
- [x] 我已合并了对后续工作无意义的commit并确认不会对后续维护造成困扰 I have merged commits that are meaningless for follow-up work and confirmed that they will not cause damage to follow-up maintenance

### CODE REVIEW

File: app/src/main/java/nil/nadph/qnotified/util/DexKit.java [hunk 1]
```diff
@@ -85,18 +84,27 @@
...
     public static final int N_BASE_CHAT_PIE__createMulti = 20004;
     public static final int N_BASE_CHAT_PIE__chooseMsg = 20005;
     public static final int N_LeftSwipeReply_Helper__reply = 20006;
-    public static final int DEOBF_NUM_N = 6;
-
-
+    // fixme
+    //public static final int N_TROOP_BASE_PIE__showNewTotalMemberCount = 20007;
+    
+    public static final int DEOBF_NUM_N = 7;
```
Comment (cinit) [inline app/src/main/java/nil/nadph/qnotified/util/DexKit.java:105]: Change to 6 plz

File: app/src/main/java/nil/nadph/qnotified/util/DexKit.java [hunk 2]
```diff
@@ -479,7 +514,7 @@ public static String c(int i) {
             case C_GroupAppActivity:
                 return new byte[][]{new byte[]{0x11, 0x6F, 0x6E, 0x44, 0x72, 0x61, 0x77, 0x65, 0x72, 0x53, 0x74, 0x61, 0x72, 0x74, 0x4F, 0x70, 0x65, 0x6E}};
             case C_IntimateDrawer:
-                return new byte[][]{new byte[]{0x49, 0x6e, 0x74, 0x69, 0x6d, 0x61, 0x74, 0x65, 0x44, 0x72, 0x61, 0x77, 0x65, 0x72}};
+                return new byte[][]{new byte[]{0x14, 0x49, 0x6e, 0x74, 0x69, 0x6d, 0x61, 0x74, 0x65, 0x44, 0x72, 0x61, 0x77, 0x65, 0x72}};
```
Comment (cinit) [inline app/src/main/java/nil/nadph/qnotified/util/DexKit.java:517]: 确认是0x14?

**Review by cinit — CHANGES_REQUESTED** [review]

File: app/src/main/java/nil/nadph/qnotified/util/DexKit.java (hunk 2 above)
Comment (ketal178) [reply app/src/main/java/nil/nadph/qnotified/util/DexKit.java:517]: 我怀疑他没用测试（

Comment (cinit) [reply app/src/main/java/nil/nadph/qnotified/util/DexKit.java:517]: @kyuubiran

Comment (KyuubiRan) [reply app/src/main/java/nil/nadph/qnotified/util/DexKit.java:517]: ?我不知道 不要问我 我不会dexkit

Comment (NextAlone) [reply app/src/main/java/nil/nadph/qnotified/util/DexKit.java:517]: 额。0x0e

Comment (ketal178) [reply app/src/main/java/nil/nadph/qnotified/util/DexKit.java:517]: 当初的0x49怎么过的（

Comment (cinit) [reply app/src/main/java/nil/nadph/qnotified/util/DexKit.java]: 可是我在8.5.0(1596)没找到字符串`"IntimateDrawer"`啊

Comment (NextAlone) [reply app/src/main/java/nil/nadph/qnotified/util/DexKit.java]: ?我不知道 不要问我 我当时不会dexkit

Comment (ketal178) [reply app/src/main/java/nil/nadph/qnotified/util/DexKit.java]: > ?我不知道 不要问我 我当时不会dexkit

神秘（

Comment (cinit) [reply app/src/main/java/nil/nadph/qnotified/util/DexKit.java]: > ?我不知道 不要问我 我当时不会dexkit

How had you done it?

Comment (NextAlone) [reply app/src/main/java/nil/nadph/qnotified/util/DexKit.java]: > How had you done it?

imitation?

Comment (NextAlone) [reply app/src/main/java/nil/nadph/qnotified/util/DexKit.java]: > 可是我在8.5.0(1596)没找到字符串`"IntimateDrawer"`啊

哦，当时不知道是字符串哦，谁知道呢。

**Review by cinit — DISMISSED** [review]

**Review by lliioollcn — APPROVED** [review]

**Review by cinit — COMMENTED** [review]
@singleNeuron sth not working

**Review by ketal178 — APPROVED** [review]

**Review by cinit — APPROVED** [review]

**Pull request merged**